app/controllers/users/folders/binaries_controller.rb

```diff
@@ -29,6 +29,12 @@ def show
     render 'users/folders/binaries/show.html.erb'
   end
 
+  def destroy
+    binary = Binary.find(params[:id])
```

Comment:
To work with the authorization scheme I implemented, you're going to have to pass username and route through params, then find the user, then the user's folder, then the binary from that folder.